assets/airports/kmco.json

```diff
@@ -0,0 +1,6135 @@
+              "KMCO35L": [["LAMMA", "A120"], "JESUP", "COZMO", "BITHO"],
+              "KMCO35R": [["LAMMA", "A120"], "JESUP", "COZMO", "BITHO"],
+              "KMCO36L": [["LAMMA", "A120"], "JESUP", "COZMO", "BITHO"],
+              "KMCO36R": [["LAMMA", "A120"], "JESUP", "COZMO", "BITHO"]
+            },
+            "draw": [
+                ["OMN*", "LAMMA", "JESUP", "COZMO", "BITHO"]
+            ]
+        },
+        "GOOFY7": {
+            "icao": "GOOFY7",
+            "name": "Goofy Seven",
+            "entryPoints": {
+              "PBI": ["PBI", "LLNCH"],
+              "PHK": ["PHK"],
+              "TRV": ["TRV"]
+            },
+            "body": [],
+            "rwy": {
+              "KMCO17L": [["BAIRN", "A110"], "ODDEL", "#360"],
+              "KMCO17R": [["BAIRN", "A110"], "ODDEL", "#360"],
+              "KMCO18L": [["BAIRN", "A110"], "ODDEL", "#360"],
+              "KMCO18R": [["BAIRN", "A110"], "ODDEL", "#360"],
+              "KMCO35L": [["BAIRN", "A80"], "#360"],
+              "KMCO35R": [["BAIRN", "A80"], "#360"],
+              "KMCO36L": [["BAIRN", "A80"], "#360"],
+              "KMCO36R": [["BAIRN", "A80"], "#360"]
+            },
+            "draw": [
+                ["TRV*", "BAIRN", "ODDEL"],
+                ["PBI*", "BAIRN"],
+                ["PHK*", "BAIRN"]
+            ]
+        },
+        "GRNCH2": {
+            "icao": "GRNCH2",
+            "name": "Grinch Two",
+            "entryPoints": {
+                "ADLYN": ["ADLYN", "ELITE", ["MAYYR", "A240+"]],
+                "CATEE": ["CATEE", "ELITE", ["MAYYR", "A240+"]],
+                "COAXE": ["COAXE"]
+            },
+            "body": ["GRTNG", "GRNCH", ["BUGGZ", "A110+|A160-|S280"], "BLAYK"],
+            "rwy": {
+                "KMCO17L": [["BUNIE", "A70|S230"], ["^YAGGO", "A60|S210"], "#107"],
+                "KMCO17R": [["BUNIE", "A70|S230"], ["^YAGGO", "A60|S210"], "#107"],
+                "KMCO18L": [["BUNIE", "A70|S230"], ["^YAGGO", "A60|S210"], "#107"],
+                "KMCO18R": [["BUNIE", "A70|S230"], ["^YAGGO", "A60|S210"], "#107"],
+                "KMCO35L": [["WHOVL", "A90+|A140-|S250"], "HARAM", ["DORII", "A80+|S220"], ["^BRICE", "A50|S210"], "#186"],
+                "KMCO35R": [["WHOVL", "A90+|A140-|S250"], "HARAM", ["DORII", "A80+|S220"], ["^BRICE", "A50|S210"], "#186"],
+                "KMCO36L": [["WHOVL", "A90+|A140-|S250"], "HARAM", ["DORII", "A80+|S220"], ["^BRICE", "A50|S210"], "#186"],
+                "KMCO36R": [["WHOVL", "A90+|A140-|S250"], "HARAM", ["DORII", "A80+|S220"], ["^BRICE", "A50|S210"], "#186"]
+            },
+            "draw": [
+                ["ADLYN*", "ELITE", "MAYYR", "COAXE*", "GRTNG", "GRNCH", "BUGGZ", "BLAYK", "BUNIE", "YAGGO"],
+                ["CATEE*", "ELITE"],
+                ["BLAYK", "WHOVL", "HARAM", "DORII", "BRICE"]
+            ]  
+        },
+        "JAFAR2": {
+            "icao": "JAFAR2",
+            "name": "Jafar Two",
+            "entryPoints": {
```

Comment:
Missing BRKWL transition

Reply:
This seems contradictory to the statement above about the COAXE fix. All transitions pass through BRKWL. So do they need to go in the body or entryPoints?